react/src/pages/user/profile/ProfilePage.jsx

```
@@ -24,32 +24,61 @@
+      }
     } catch (error) {
       console.error('Failed to update password:', error);
     }
   };
 
   if (isLoading) {
-    return <div className={styles.loading}>{t('profile.loading')}</div>;
+    return (
+      <div className={styles.profileContainer}>
+        <div className={styles.loading}>
+          <div className={styles.spinner}></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)